student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom, _name = 'unknown')
+    super('sharon', 20)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly notice that in the `Student` constructor you should send the data for the `Person` class constructor as it is its parent class.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```